mission_control/navigator_missions/nav_missions/start_gate.py

```diff
@@ -1,66 +1,277 @@
 #!/usr/bin/env python
+from __future__ import division
+
 import txros
 import tf
+import tf.transformations as trns
 import numpy as np
 import navigator_tools
+from sensor_msgs.msg import PointCloud
 from twisted.internet import defer
+from navigator_msgs.srv import ObjectDBSingleQuery, ObjectDBSingleQueryRequest
+
+
+class Buoy(object):
+    @classmethod
+    def from_srv(cls, srv):
+        obj = srv.object
+        return cls(obj.position, "Black")
+
+    def __init__(self, position, color):
+        self.position = np.array(position)
+        self.color = color
+        self.odom = np.array([0, 0, 0])
+
+    def __repr__(self):
+        return "BUOY at: {} with color {}".format(self.position, self.color)
+
+    def distance(self, odom):
+        dist = np.linalg.norm(self.position - odom)
+        #print dist
+        return dist
+
+
+def get_buoys():
```

Comment:
Once that system is all set up yes - this `get_buoys` is used to simulated the buoys being detected without having the database up and running (which requires a lot of other backend)